tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/HelloWorldTool/HelloWorldTool.cs

```diff
@@ -44,6 +44,7 @@ public override async Task HandleCommand(InvocationContext ctx, CancellationToke
         }
 
         [McpServerTool(Name = "hello-world-fail"), Description("Echoes the message back to the client with a failure")]
+        #pragma warning disable MCP001
```

Comment:
Which warning is this?

Reply:
Oh, it's ours. I see.

Reply:
We should probably wrap this tool anyway since it's ideally supposed to be instructive

Reply:
Done.